[2024-08-06T14:24:26.615+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Credit_score_pipeline_v04.load_data manual__2024-08-06T14:05:59.245755+00:00 [queued]>
[2024-08-06T14:24:26.628+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Credit_score_pipeline_v04.load_data manual__2024-08-06T14:05:59.245755+00:00 [queued]>
[2024-08-06T14:24:26.630+0000] {taskinstance.py:2193} INFO - Starting attempt 7 of 12
[2024-08-06T14:24:26.662+0000] {taskinstance.py:2214} INFO - Executing <Task(PostgresOperator): load_data> on 2024-08-06 14:05:59.245755+00:00
[2024-08-06T14:24:26.672+0000] {standard_task_runner.py:60} INFO - Started process 686 to run task
[2024-08-06T14:24:26.676+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Credit_score_pipeline_v04', 'load_data', 'manual__2024-08-06T14:05:59.245755+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/credit_score.py', '--cfg-path', '/tmp/tmpd_9dxoy7']
[2024-08-06T14:24:26.682+0000] {standard_task_runner.py:88} INFO - Job 196: Subtask load_data
[2024-08-06T14:24:26.749+0000] {task_command.py:423} INFO - Running <TaskInstance: Credit_score_pipeline_v04.load_data manual__2024-08-06T14:05:59.245755+00:00 [running]> on host 43471a75e062
[2024-08-06T14:24:26.855+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Credit_score_pipeline_v04' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-06T14:05:59.245755+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-06T14:05:59.245755+00:00'
[2024-08-06T14:24:26.856+0000] {sql.py:276} INFO - Executing: 
       COPY credit FROM '/opt/***/dags/clean_train.csv' WITH (FORMAT csv, HEADER true, DELIMITER ',');

        
[2024-08-06T14:24:26.869+0000] {base.py:83} INFO - Using connection ID 'postgresql' for task execution.
[2024-08-06T14:24:26.882+0000] {base.py:83} INFO - Using connection ID 'postgresql' for task execution.
[2024-08-06T14:24:26.921+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 397, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/postgres/hooks/postgres.py", line 161, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "host.docker.internal" (192.168.65.254), port 5433 failed: FATAL:  password authentication failed for user "postgres"

[2024-08-06T14:24:26.934+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=Credit_score_pipeline_v04, task_id=load_data, execution_date=20240806T140559, start_date=20240806T142426, end_date=20240806T142426
[2024-08-06T14:24:26.950+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 196 for task load_data (connection to server at "host.docker.internal" (192.168.65.254), port 5433 failed: FATAL:  password authentication failed for user "postgres"
; 686)
[2024-08-06T14:24:27.019+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-08-06T14:24:27.043+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
